src/components/dashboard/Reason.js

```diff
@@ -33,19 +82,167 @@ const SendReason = (props: AmountProps) => {
+                }}
+              >
+                <CategoryBox
+                  style={[
+                    styles.categoryBox,
+                    styles.border,
+                    screenState.category === PaymentCategory.Product ? { borderWidth: 2 } : { borderWidth: 0 },
+                  ]}
+                  title={PaymentCategory.labelOf(PaymentCategory.Product)}
+                >
+                  <View>
+                    <ProductSVG />
+                  </View>
+                </CategoryBox>
+              </TouchableOpacity>
+            </Section.Stack>
+          </Section.Row>
+          <Section.Row style={[styles.categoryRow, { paddingTop: theme.sizes.default }]}>
+            <Section.Stack>
+              <TouchableOpacity
+                onPress={() => {
+                  handleCategoryBoxOnPress(PaymentCategory.Course)
+                }}
+              >
+                <CategoryBox
+                  style={[
+                    styles.categoryBox,
+                    styles.border,
+                    screenState.category === PaymentCategory.Course ? { borderWidth: 2 } : { borderWidth: 0 },
+                  ]}
+                  title={'Course / Private\nConsultation'}
+                >
+                  <View>
+                    <CourseSVG />
+                  </View>
+                </CategoryBox>
+              </TouchableOpacity>
+            </Section.Stack>
+            <Section.Stack>
+              <TouchableOpacity
+                onPress={() => {
+                  handleCategoryBoxOnPress(PaymentCategory.Donation)
+                }}
+              >
+                <CategoryBox
+                  style={[
+                    styles.categoryBox,
+                    styles.border,
+                    screenState.category === PaymentCategory.Donation ? { borderWidth: 2 } : { borderWidth: 0 },
+                  ]}
+                  title={PaymentCategory.labelOf(PaymentCategory.Donation)}
+                >
+                  <View>
+                    <DonationSVG />
+                  </View>
+                </CategoryBox>
+              </TouchableOpacity>
+            </Section.Stack>
+            <Section.Stack>
+              <TouchableOpacity
+                onPress={() => {
+                  handleCategoryBoxOnPress(PaymentCategory.Other)
+                }}
+              >
+                <CategoryBox
+                  style={[
+                    styles.categoryBox,
+                    styles.border,
+                    screenState.category === PaymentCategory.Other ? { borderWidth: 2 } : { borderWidth: 0 },
+                  ]}
+                  title={PaymentCategory.labelOf(PaymentCategory.Other)}
+                >
+                  <View>
+                    <OtherSVG />
+                  </View>
+                </CategoryBox>
+              </TouchableOpacity>
+            </Section.Stack>
+          </Section.Row>
+
```

Comment:
this should be renderd using .map
see for example how we render social buttons in invite.js

Reply:
@omerzam

Reply:
what is the benefit of this?
also, there are too many differences, different SVG asset for each box, and different rows

Reply:
@sirpy

Reply:
please look at the social buttons and do the same, we have different SVG/icon there also.
the purpose is that it will be easy to modify and reason about.
also you can do it in one row with wrapping
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items